src/createSynaptik.tsx

```diff
@@ -17,6 +18,10 @@ export type StoreMap<T extends ConstructorMap> = {
   [P in keyof T]: InstanceType<T[P]>;
 };
 
+export type ProviderProps = {
+  children?: ReactNode;
```

Comment:
Should this be `ReactNode| ReactNode[]` ?

Reply:
Not sure but [seeing this type definition from @types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L230) I would say `ReactNode` should be enough both others are already included on it